sql/core/src/test/scala/org/apache/spark/sql/execution/streaming/sources/RateStreamProviderSuite.scala

```diff
@@ -319,29 +307,18 @@ class RateSourceSuite extends StreamTest {
       "rate source does not support user-specified schema"))
   }
 
-  test("continuous in registry") {
```

Comment:
we don't need this test now. With the new `Format` abstraction, the lookup logic is unified between microbatch and continuous